src/Symfony/Installer/NewCommand.php

```diff
@@ -350,6 +350,18 @@ protected function updateComposerJson()
             unset($contents['description']);
         }
 
+        if (isset($contents['config']['platform']['php'])) {
+            unset($contents['config']['platform']['php']);
+
+            if (empty($contents['config']['platform'])) {
+                unset($contents['config']['platform']);
+            }
+
+            if (empty($contents['config'])) {
```

Comment:
I think this does not complexify the behavior, it just simplifies the outputted JSON